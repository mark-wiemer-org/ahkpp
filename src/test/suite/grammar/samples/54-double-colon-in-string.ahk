;[#54 in AHK+](https://github.com/vscode-autohotkey/autohotkey-plus/issues/54)
Hotstring("::ykhis", "you know how it is")
Hotstring(":C:OOS", "out-of::-spec")