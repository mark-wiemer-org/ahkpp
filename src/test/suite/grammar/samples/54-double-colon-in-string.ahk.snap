>;[#54 in AHK+](https://github.com/vscode-autohotkey/autohotkey-plus/issues/54)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>Hotstring("::ykhis", "you know how it is")
#^^^^^^^^^ source.ahk entity.name.function.ahk
#         ^ source.ahk punctuation.bracket.ahk
#          ^ source.ahk string.quoted.double.ahk punctuation.definition.string.begin.ahk
#           ^^^^^^^ source.ahk string.quoted.double.ahk
#                  ^ source.ahk string.quoted.double.ahk punctuation.definition.string.end.ahk
#                   ^ source.ahk punctuation.ahk
#                    ^ source.ahk
#                     ^ source.ahk string.quoted.double.ahk punctuation.definition.string.begin.ahk
#                      ^^^^^^^^^^^^^^^^^^ source.ahk string.quoted.double.ahk
#                                        ^ source.ahk string.quoted.double.ahk punctuation.definition.string.end.ahk
#                                         ^ source.ahk punctuation.bracket.ahk
>Hotstring(":C:OOS", "out-of::-spec")
#^^^^^^^^^ source.ahk entity.name.function.ahk
#         ^ source.ahk punctuation.bracket.ahk
#          ^ source.ahk string.quoted.double.ahk punctuation.definition.string.begin.ahk
#           ^^^^^^ source.ahk string.quoted.double.ahk
#                 ^ source.ahk string.quoted.double.ahk punctuation.definition.string.end.ahk
#                  ^ source.ahk punctuation.ahk
#                   ^ source.ahk
#                    ^ source.ahk string.quoted.double.ahk punctuation.definition.string.begin.ahk
#                     ^^^^^^^^^^^^^ source.ahk string.quoted.double.ahk
#                                  ^ source.ahk string.quoted.double.ahk punctuation.definition.string.end.ahk
#                                   ^ source.ahk punctuation.bracket.ahk